packages/devtools_app/lib/src/screens/vm_developer/vm_object_model.dart

```diff
@@ -208,7 +209,18 @@ class ScriptObject extends VmObject {
   SourceLocation? get _sourceLocation => null;
 
   @override
-  String? get name => null;
+  String? get name => (obj.uri ?? scriptRef?.uri)?.split('/').last;
+
+  late final DateTime? loadTime;
+
+  @override
+  Future<void> initialize() async {
+    await super.initialize();
+
+    if (obj.loadTime != null) {
+      loadTime = DateTime.fromMillisecondsSinceEpoch(obj.loadTime!);
```

Comment:
This should be relatively cheap to create, so I think we can just update `loadTime` to be a getter that does this operation. That way we can completely remove the need for an override of `initialize()`, at least for now.

Reply:
Done